src/structures/Message.js

```diff
@@ -90,6 +90,16 @@ class Message extends Base {
          */
         this.hasQuotedMsg = data.quotedMsg ? true : false;
 
+        /**
+         * Indicates the mentions in the message body.
+         * @type {[id]}
+         */
+        this.mentions = [];
```

Comment:
Rename `mentions` to `mentionedIds` to indicate the difference between this and the other function.